src/esi/Libxml2Parser.cc

```diff
@@ -16,8 +16,30 @@
 
 #if USE_SQUID_ESI && HAVE_LIBXML2
 
+#include "base/RunnersRegistry.h"
 #include "esi/Libxml2Parser.h"
 
+#include <memory>
+
+namespace Esi
+{
+
+class Xml2Rr : public RegisteredRunner
+{
+public:
+    void finalizeConfig()
+    {
+        prLibxml.reset(new ESIParser::Register("libxml2", &ESILibxml2Parser::NewParser));
+    }
+
+private:
+    std::unique_ptr<ESIParser::Register> prLibxml;
```

Comment:
This data member should be called prLibxml2 for consistency sake. However, to avoid "foo inside Foo" noise it is better to rename this member and the matching ExpatRr::prExpat member to `registration` or a similar library-neutral purpose-focused name.

Reply:
Done in cbc3ab2